src/crab/ebpf_transformer.cpp

```
@@ -257,6 +259,10 @@ void EbpfTransformer::operator()(const Assume& s) {
 
 void EbpfTransformer::operator()(const Undefined& a) {}
 
+void EbpfTransformer::operator()(const CallBtf&) {}
```

Clang (14.0.6)

[warning] all parameters should be named in a function

(readability-named-parameter)